Sources/EpoxyLayoutGroups/Extensions/Alignment+Equatable.swift

```diff
@@ -0,0 +1,52 @@
+// Created by Tyler Hedrick on 6/8/20.
+// Copyright © 2020 Airbnb Inc. All rights reserved.
+
+import UIKit
+
+// MARK: - VGroup.ItemAlignment + Equatable
+
+extension VGroup.ItemAlignment: Equatable {
```

Comment:
I think I mentioned this to you a while back but it'd be nice to get `Hashable` conformances here too so we can include these in `Style`s.

Reply:
they are hashable!

Reply:
I guess I can remove this Equatable conformance 😅

Reply:
I ended up moving these item alignments into their own files to put Equatable and Hashable conformances all in one spot and I think it's much cleaner that way 👍